cost-analyzer/templates/kubecost-oidc-secret-template.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.oidc }}
+{{- if .Values.oidc.secretName }}
```

Comment:
Instead of `if` checks here and silently failing to create the secret, we can use  the `required` function on fields that are required when OIDC is enabled https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function

Reply:
That's cool, didn't know you could do that.

In this case though I think it's fine, technically you can create the secret yourself, so if there's an issue there it'll fail nonsilently elsewhere, either within the cost-model or in the volume mount.